Admin-Guide.md

```diff
@@ -0,0 +1,52 @@
+# Admin Guide
+
+## Table of Contents
+
+- [Navigating to the Administration Page](#navigating-to-the-administration-page)
+- [Managing Courses](#managing-courses)
+
+## Navigating to the Administration Page
+
+After logging in to MarkUs as an Admin, you will be directed to the MarkUs Administration home page.
+
+![Markus Admin Dashboard](images/markus-admin-dashboard.png)
+
+If you navigate away from the Administration Page and wish to return, click on the "Administration" tab found in the header.
+
+![Markus Admin Tab](images/markus-admin-tab.png)
```

Comment:
Will not add an image until role switching is implemented in order to show where this tab is in relation to other header tabs.